ceph_deploy/mon.py

```diff
@@ -398,11 +395,7 @@ def mon_destroy(args):
 
 
 def mon_create_initial(args):
-    cfg = conf.ceph.load(args)
-    cfg_initial_members = cfg.safe_get('global', 'mon_initial_members')
-    if cfg_initial_members is None:
-        raise RuntimeError('No `mon initial members` defined in config')
-    mon_initial_members = re.split(r'[,\s]+', cfg_initial_members)
+    mon_initial_members = get_mon_member(args, True)
```

Comment:
Since the second argument to get_mon_member is a kwarg, I'd rather see this as get_mon_member(args, init_tags=True).